### ISSUE TITLE
accepted version

### ISSUE THREAD

**Issue by MarcelDevG** [dotnet/docs#5554]
from the compiler:
/langversion:<number>             Specify language version:
                                  9|9.0|10|10.0|11|11.0|12|12.0|14|14.0

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 98333251-2a19-793a-acd2-274181a7f1fd
* Version Independent ID: ff27427e-cbe2-3d89-b526-dfa7b3debabf
* Content: [-langversion (Visual Basic)](https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/command-line-compiler/langversion)
* Content Source: [docs/visual-basic/reference/command-line-compiler/langversion.md](https://github.com/dotnet/docs/blob/master/docs/visual-basic/reference/command-line-compiler/langversion.md)
* Service: **unspecified**
* Product: **dotnet-visualbasic**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by svick** [comment]
I agree that the page should be updated, though it seems there are versions beyond 14. When I run `vbc /langversion:?` from Developer Command Prompt for VS 2017, the output I get is:

```
Supported language versions:
default
9
10
11
12
14
15 (default)
15.3
15.5 (latest)
latest
```

Do you want to create a Pull Request to update that information? If you do and don't know how to do that, you might want to read [this guide](https://docs.microsoft.com/en-us/contribute/#quick-edits-to-existing-documents). And if you have any questions about the process, feel free to ask here.

**Comment by MarcelDevG** [comment]
Aha, there was an older version of vbc in my path. Your list is correct. I will dive into the guide.

**Comment by mairaw** [comment]
Resolved with PR #5571
